.travis.yml

```diff
@@ -26,10 +26,7 @@ before_install:
 - php --version
 - composer install
 script:
-- npm run force-build
-- npm run test-base
-- npm run test-base-ws
-- pip list
```

Comment:
i assume this was not removed accidentally, but we no longer need that command, right?

Reply:
Yeah pip list was only listing the installed packages and occupying space...